Makefile

```diff
@@ -6,14 +6,14 @@ help:
 	@echo "    clean"
 	@echo "        Remove python artifacts and build artifacts."
 	@echo "    lint"
-	@echo "        Check style with flake8."
+	@echo "        Check style with flakehell."
```

Comment:
what is the reason to switch to `flakehell`? it doesn't seem to be widely used and is very new. I'd try to avoid using it, even if that means keeping some of the configuration in `setup.cfg`

Reply:
It's just a wrapper for flake8, but you're right — we can remove it and keep using flake8